projects/arc-lib/src/lib/components/auth/login/login.component.scss

```diff
@@ -19,23 +21,86 @@
   }
 }
 
-.google-btn {
-  width: 24.125rem;
-  height: 4rem;
-  border-radius: 0.375rem;
-  font-size: 1.25rem;
-  font-family: #{nb-theme(font-family-primary)};
-  font-weight: 600;
-  line-height: 1.5rem;
-  background-color: map.get($font, "light");
-  border: 0.063rem solid map.get($color, "border-disable");
-  position: absolute;
+.card-row {
+  display: flex;
+  margin: 0 -0.5rem;
+}
 
-  &:hover {
-    border-color: map-get($font, "placeholder");
+.card-col {
+  flex: 1 0 calc(50% - 1rem);
+  margin: 0 0.5rem;
+}
+.card-row {
+  margin: 0;
+  padding: 30px 70px;
+  .sign-in-title {
+    h2 {
+      font-size: 36px;
+      margin: 50px 0 10px 0;
+    }
+    h3 {
+      font-size: 24px;
+      color: #525252;
+      margin: 0;
+    }
+    p {
+      font-size: 16px;
+      margin: 0 0 30px 0;
+    }
   }
+}
 
-  &:active {
-    border-color: map-get($font, "primary");
+.sign-in-button {
+  button {
+    font-weight: normal;
+    border: 1px solid #d9d9d9;
+    background: #fff;
+    text-transform: inherit;
+    font-size: 13px;
+    &:hover {
+      border: 1px solid #8f8f8f;
+    }
+    img {
+      margin-right: 5px;
+    }
   }
 }
+.footer {
+  color: #525252;
+  a {
+    text-decoration: none;
+    color: #19a5ff;
+    font-weight: 700;
```

Comment:
same why repeating css can we move this to base module/component /auth

Reply:
i make the same name (.sign-up-button ) and move this code to Main styles.scss file n apply this name to every component where we are using this code